doc/api.md

```diff
@@ -266,6 +266,8 @@ the draft content store with the published item, if one exists. Uses
 
  [Request/Response detail](https://pact-broker.dev.publishing.service.gov.uk/pacts/provider/Publishing%20API/consumer/GDS%20API%20Adapters/latest#a_get_entries_request_given_a_content_item_exists_in_multiple_locales_with_content_id:_bed722e6-db68-43e5-9079-063f623335a7)
 
+⚠️ **This endpoint will only return content by the authenticating app, unless that app's API key has `view_all` permissions.** 
```

Comment:
Rather than saying "by the", I think it would be better to use the same terminology as here [1], which would mean saying something like:
```This endpoint will only return content for which the value of `publishing_app` matches the name of the application making the request, unless that app's API key has `view_all` permissions```

1: https://github.com/alphagov/publishing-api/blob/master/doc/model.md#publishing_app